resources/js/components/shared/ModalSaveVersion.vue

```diff
@@ -528,8 +535,20 @@ export default {
         });
       } else {
         this.saveFromEditLaunchpad();
+        dataProcess = this.process;
+        dataProcess.description = this.processDescription;
       }
 
+      //Save only process description field using Process API
+      this.process.description = this.processDescription;
+      ProcessMaker.apiClient.put('processes/' + this.options.id, dataProcess)
+        .then(response => {
+          ProcessMaker.alert(this.$t('The process was saved.'), 'success', 5, true);
+        })
+        .catch(error => {
+          console.log('Error: ', error);
```

Comment:
use console.error instead of console.log

Reply:
solved